specification/metrics/semantic_conventions/README.md

```diff
@@ -125,6 +126,22 @@ to the processes then to represent the count of the processes we can have a metr
 `system.processes.count`. The suffix `count` here indicates that it is the count of
 `system.processes`.
 
+This rule SHOULD only be applied to UpDownCounters, since (monotonic) Counters have
+`_total` appended to their names when they are
+[mapped to Prometheus](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/compatibility/prometheus_and_openmetrics.md#otlp-metric-points-to-prometheus),
+which would lead to `_count_total`.
+
+#### Do not use `total`
+
+Counters SHOULD NOT append `_total`  to their names. Counters have `_total` appended to their names when they are
+[mapped to Prometheus](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/compatibility/prometheus_and_openmetrics.md#otlp-metric-points-to-prometheus),
+which would lead to `_total_total`. The reason that the Prometheus mapping cannot
+suppress adding the duplicate `_total` is because then the mapping wouldn't be
```

Comment:
The prometheus receiver likely won't remove _total suffixes going forward, so this won't apply.  See https://github.com/open-telemetry/wg-prometheus/issues/72.

Reply:
If system.processes.count is an UpDownCounter (which I expect it should be), then it is represented as a gauge in Prometheus.  In that case, it won't have _total appended.

Reply:
@dashpole PTAL.